src/client/common/components/base-network-view/base-network-view.js

```diff
@@ -218,7 +224,7 @@ class BaseNetworkView extends React.Component {
         ]
       ),
       h('div', {
-        className: classNames('sidebar-menu', { 'sidebar-menu-open': this.state.open })
+        className: classNames('sidebar-menu',{'sidebar-menu-small':this.state.activeMenu === 'filter-menu',  'sidebar-menu-open': this.state.open })
```

Comment:
same here:
- dont use a classname and instead inject the width that you want the menu to be through props